aws_lambda_decorators/validators.py

```diff
@@ -85,34 +99,23 @@ def validate(self, value=None):
             value (object): Value to be validated.
         """
         try:
-            return self._schema.validate(value) == value
+            return self._condition.validate(value) == value
         except SchemaError:
             return False
 
-    def message(self, value=None):
-        """
-        Gets the formatted error message for a failed schema check
 
-        Args:
-            value (dict): The validated value
-
-        Returns:
-            The error message
-        """
-        return f"{value} does not validate against schema {self._schema}"
-
-
-class Minimum:  # noqa: pylint - too-few-public-methods
+class Minimum(Validator):  # noqa: pylint - too-few-public-methods
     """Validation rule to check if a value is greater than a minimum value."""
 
-    def __init__(self, minimum: (float, int)):
+    def __init__(self, minimum: (float, int), error_message=None):
         """
         Set the minimum value.
 
         Args:
             minimum (float, int): The minimum value.
+            error_message (str): A custom error message to output if validation fails
         """
-        self._minimum = minimum
+        Validator.__init__(self, error_message or "'{value}' is smaller than minimum value '{condition}'", minimum)
```

Comment:
Trivial but maybe change "smaller than" to "less than".

Reply:
made the changes. Could you check again please?